ruleset.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<ruleset name="Google-Cloud-PHP-PSR2">
+  <rule ref="PSR2">
+    <!--
+      Exclude this sniff because we override methods with underscores in some classes
+    -->
+    <exclude name="PSR2.Methods.MethodDeclaration.Underscore"/>
```

Comment:
The generated code also includes other names that need to be excluded here, in particular these errors:

* `Method name "PhotosLibraryGrpcClient::CreateAlbum" is not in camel caps format` (and related)
* `Class name "FeatureFilter_Feature" is not in PascalCase format`

The current exclude doesn't seem to cover these.

Reply:
I think they are covered with this `exclude-pattern`?

https://github.com/arielmagbanua/php-photoslibrary/blob/338f460165ed9ef5f71f9d3d420ab71f0094e3b7/phpcs_ruleset.xml#L11

Reply:
Actually, there are a few handwritten files in there - unfortunately there isn't an easy way to distinguish between them using this syntax. For now I think it's okay to merge this in like this, while we come up with a better way to handle the exclude pattern. (Short of moving some classes into different packages, we may have to include a longer list of excluded files here.)